step.yml

```diff
@@ -37,46 +55,45 @@ inputs:
       is_required: true
   - keystore_url: $BITRISEIO_ANDROID_KEYSTORE_URL
     opts:
-      title: "Keystore url"
-      summary: ""
+      title: "Keystore URL"
+      summary: "The keystore file's URL which is generated when you upload the file to the Code Signing tab."
       description: |-
         TO BE FILLED
       is_required: true
       is_sensitive: true
   - keystore_password: $BITRISEIO_ANDROID_KEYSTORE_PASSWORD
     opts:
       title: "Keystore password"
-      summary: ""
+      summary: "The password you added to the keystore."
       description: ""
       is_required: true
       is_sensitive: true
   - keystore_alias: $BITRISEIO_ANDROID_KEYSTORE_ALIAS
     opts:
       title: "Keystore alias"
-      summary: ""
+      summary: "Identifier name you added to the keystore."
       description: ""
       is_required: true
       is_sensitive: true
   - private_key_password: $BITRISEIO_ANDROID_KEYSTORE_PRIVATE_KEY_PASSWORD
     opts:
       title: "Private key password"
-      summary: ""
-      description: |
-        If key password equals to keystore password, you can leave it empty.
-        Otherwise specify the private key password.
+      summary: "Password you added to the private key."
+      description:
       is_sensitive: true
-  - bundletool_version: "1.0.0"
+  - bundletool_version: "0.15.0"
```

Comment:
Please use "1.0.0" for the bundletool_version as we have updated to latest version on master.